app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_secure_password
+
+  validates :name, presence: true
+  validates :email, format: { with: /\S+@\S+/ }, uniqueness: { case_sensitive: false }
```

Comment:
Rails has a default REGEX for matching emails. You can use this.

```suggestion
  validates :email, format: { with: URI::MailTo::EMAIL_REGEXP }, uniqueness: { case_sensitive: false }
```

Reply:
@sku-wtag bhai, I have updated it.